docusaurus.config.ts

```diff
@@ -1,12 +1,12 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+import {themes as prismThemes} from 'prism-react-renderer';
+import type { Config, ReportingSeverity } from '@docusaurus/types';
+import type * as Preset from '@docusaurus/preset-classic';
```

Comment:
Consider importing only the necessary types
```suggestion
import type { Option, ThemeConfig } from '@docusaurus/preset-classic';
```